Chapter_2/ex2.rb

```diff
@@ -0,0 +1,11 @@
+#! usr/bin/env ruby
+
+# A comment, this is so you can read your comment later
```

Comment:
We're practicing `attention to detail` and `noticing differences` with this assignment.  How is your line different that the line in the chapter?